### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by VeenDuco**
Suggested code of conduct template by Github applied to our project. Reporting procedures made such that it is know who will review any reports and where reports against those individuals can be filed.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+asreview.codeofconduct@uu.nl which is visible to @J535D165, @Rensvandeschoot and @VeenDuco. Regarding complaints against these individuals please use procedures of Utrecht University, see [here](https://www.uu.nl/en/organisation/about-us/codes-of-conduct/inappropriate-behaviour).
```
Comment (J535D165) [inline CODE_OF_CONDUCT.md:63]: A hard wrap here would be nice to have. This makes the line readable in text format on smaller screens.

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks for proposing this. We will merge after the comment is resolved.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Rensvandeschoot) [reply CODE_OF_CONDUCT.md]: done

**Review by Rensvandeschoot — APPROVED** [review]

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Thanks

**Pull request merged**